codebuild/release/test-release.yml

```diff
@@ -9,17 +9,35 @@ env:
 
 phases:
   install:
+    commands:
+      - pip install tox
+      - pip install --upgrade pip
     runtime-versions:
       python: latest
+  pre_build:
+    commands:
+      - git checkout $COMMIT_ID
+      - FOUND_VERSION=$(sed -n 's/__version__ = "\(.*\)"/\1/p' src/aws_encryption_sdk/identifiers.py)
+      - |
+        if expr ${FOUND_VERSION} != ${VERSION}; then
+          echo "identifiers.py does not contain expected version string ${VERSION}, stopping"
+          exit 1;
+        fi
   build:
     commands:
-      - pip install tox
-      - git checkout $BRANCH
       - tox -e park
       - tox -e test-release
 
 
 batch:
-  fast-fail: false
-  build-list:
+  fast-fail: true
+  build-graph:
     - identifier: test_release
```

Comment:
These labels are a little confusing: in isolation I'm not sure if this one means "test the release" or "release the test artifact". Perhaps we can use phrases like "release to production" vs "release to staging"?

Reply:
Makes sense, will update wording.